src/view/userCommands.java

```diff
@@ -0,0 +1,246 @@
+package view;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+import controller.*;
+
+
+public class userCommands {
+
+    //TODO: While MVCing - split to userCommands interface, userCommands abs class then mazeUserCommands
+    private ArrayList<maze2d> generatedMazes = new ArrayList<>();
```

Comment:
Something that came to mind, usually class members are starting with a prefix so it will be easier to find them in class function implementations, just so you'd know, you can add a m_generatedMazes or as I prefer _generatedMazes and keep it consistent